### PR TITLE
Nerfs Beekit + Beesword

### PR DESCRIPTION
**Pull request by wesoda25**
## About The Pull Request

Beekit now gets one less beenade and has a lower chance, beesword has worse armor pen, does 2 less damage and injects 4 toxin as opposed to 4 histamine (could be looked at as a buff though). Plus I removed some pointless // remarks that I previously made but I doubt that really matters

## Why It's Good For The Game

Pretty overpowered, hopefully this will make things better. 

## Changelog
:cl:
balance: Beekit now has a lesser chance of spawning and has one less grenade
balance: beesword now does 2 less damage, has worse armor penetration, and injects toxin as opposed to histamine
/:cl:

### CODE REVIEW

**Comment by vuonojenmustaturska** [comment]
`i really fucking hate bee memes and I really wanna remove this stuff but whatever` why not PR a revert?

**Comment by wesoda25** [comment]
Because as much as I hate the memes, I can acknowledge that bee bundle is not half bad and fits the "gimmicky" aspect of syndicate special kits. If I removed it I feel the special kit might suffer.

**Comment by Dawson1917** [comment]
This isn't very hip of you.

**Comment by wesoda25** [comment]
Actually hold on I might just make beesword inject normal toxin, but a lot of it. Overall I think that an ideal niche for beesword would be a weapon that doesn’t let you disengage if you are fighting it. Basically, if fighting someone with beesword it’s important to finish the fight as soon as possible. If you’ve been hit a few times, every second means more damage for you etc etc.

**Comment by HomocidlePancakes** [comment]
> and injects 3 amatoxin

Won't the liver purge that instantly, resulting in 0 ticks and 0 damage from poison?

**Comment by wesoda25** [comment]
Unless im mistaken I don’t think that’s how liver works

**Comment by HomocidlePancakes** [comment]
If I remember correctly it instantly purges any toxin that has 3 or less units

**Comment by nemvar** [comment]
Good stuff. Doing gods work. I have been saying from the start that it should have melee cooldown range instead of rapid. Would certainly make it more unique than armor piercing GotNS smh.

File: code/game/objects/items/storage/uplink_kits.dm [hunk 1]
```diff
@@ -168,15 +168,16 @@
 			new /obj/item/assembly/signaler(src) // 0 tc
 			new /obj/item/storage/toolbox/syndicate(src) // 1 tc
 			new /obj/item/pen/edagger(src)
-			new /obj/item/gun/energy/wormhole_projector(src) //mooorttyyyy
```
Comment (ATH1909) [inline code/game/objects/items/storage/uplink_kits.dm]: undocumented change smh

File: code/game/objects/items/storage/uplink_kits.dm [hunk 2]
```diff
@@ -168,15 +168,16 @@
...
 			new /obj/item/gun/energy/decloner/unrestricted(src)
 
 		if("bee")
 			new /obj/item/paper/fluff/bee_objectives(src) // 0 tc (motivation)
 			new /obj/item/clothing/suit/hooded/bee_costume(src) // 0 tc
 			new /obj/item/clothing/mask/rat/bee(src) // 0 tc
 			new /obj/item/storage/belt/fannypack/yellow(src) // 0 tc
-			new /obj/item/storage/box/syndie_kit/bee_grenades(src) // 15 tc
+			new /obj/item/grenade/spawnergrenade/buzzkill(src)
+			new /obj/item/grenade/spawnergrenade/buzzkill(src)
```
Comment (nemvar) [inline code/game/objects/items/storage/uplink_kits.dm]: You can probably safely remove the bee grenade box completely and please add comments comments that tell you the TC calculation, might sound inane, but it's honestly fairly important to keep such stuff maintained.

**Comment by ATH1909** [comment]
While I do like the change of the toxin in the beesword to something other than histamine, I don't like the other changes that this PR adds.

File: code/game/objects/items/storage/uplink_kits.dm (hunk 2 above)
Comment (wesoda25) [reply code/game/objects/items/storage/uplink_kits.dm]: Nukies use buzzkill box I didn’t add it in the first place

File: code/game/objects/items/storage/uplink_kits.dm (hunk 1 above)
Comment (wesoda25) [reply code/game/objects/items/storage/uplink_kits.dm]: Ok *diot

**Comment by Krysonism** [comment]
Keep the histamine and lower the amount if needed, it is the only thing that makes sense.

**Comment by wesoda25** [comment]
> Keep the histamine and lower the amount if needed, it is the only thing that makes sense.

I disagree, normal bees in code inject toxin and I plan to make the beesword inject toxin.

**Comment by wesoda25** [comment]
K made it inject 4 units of toxin (once metabolized that results in about 15 toxin damage, on top of the 5 brute - course this is all hypothetical I didn't really bother testing it which ill do so if maintainers ask), and it has worse armor pen.

**Comment by Krysonism** [comment]
> 
> 
> > Keep the histamine and lower the amount if needed, it is the only thing that makes sense.
> 
> I disagree, normal bees in code inject toxin and I plan to make the beesword inject toxin.

generic toxin is lame and boring and histamine is actually a component of bee venom and is also released during allergic reactions to stings.

**Comment by wesoda25** [comment]
Whether its cool or more realistic really doesn't change my mind, this is the best balanced I think.

**Comment by lolman360** [comment]
make it 1u venom and 3u toxin

**Comment by HomocidlePancakes** [comment]
>make it 1u venom and 3u toxin

>If I remember correctly it instantly purges any toxin that has 3 or less units

Bruh

**Comment by wesoda25** [comment]
> make it 1u venom and 3u toxin

I don’t want stacking toxin damage, I want a steady rate of slow toxin damage

**Comment by ExcessiveUseOfCobblestone** [comment]
isn't op the person who ADDED the bee stuff?

c-c-c-c-combo cringe

**Comment by wesoda25** [comment]
cobby acting like he doesn’t have regrets either

**Comment by wesoda25** [comment]
Naksu is the hero we deserve

**Comment by wesoda25** [comment]
He may be a hero but hes not a dedicated one, back open since its not getting removed.

**Pull request merged**